src/transformation/visitors/helpers/multi.ts

```diff
@@ -52,102 +40,6 @@ export function transformMultiHelperReturnStatement(
     return lua.createReturnStatement(expressions, statement);
 }
 
-function transformMultiHelperCallArguments(
-    context: TransformationContext,
-    expression: ts.CallExpression
-): lua.Expression[] | lua.Expression {
-    if (!isMultiHelperCallSignature(context, expression)) {
-        return context.transformExpression(expression);
-    }
-
-    if (expression.arguments.length === 0) {
-        return lua.createNilLiteral(expression);
-    }
-
-    return expression.arguments.map(e => context.transformExpression(e));
-}
-
-export function transformMultiHelperVariableDeclaration(
```

Comment:
Unless I'm missing something, in addition to just removing `const [a, b] = $multi(1, 2)` (which was discussed in https://github.com/TypeScriptToLua/TypeScriptToLua/pull/821#discussion_r400049568) it also drops support for `const [a, b] = unpack([1, 2])` (and other functions returning multiple values)